src/Server/Service/TokenService.php

```diff
@@ -196,16 +198,16 @@ protected function validateTokenScopes(array $scopes)
      */
     private function compareStrings($expected, $actual)
     {
-        $expected     = (string) $expected;
-        $actual       = (string) $actual;
+        $expected = (string) $expected;
+        $actual   = (string) $actual;
 
         if (function_exists('hash_equals')) {
```

Comment:
This is PHP 5.6+, we can assume that we always have it now (in all cases min dependency will be either 5.6 or 7) so the "compareStrings" can be removed and the code only use hash_equals.